raiden/network/upnpsock.py

```diff
@@ -8,6 +9,17 @@
 log = slogging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+def valid_ip_v4(address):
+    try:
+        ipaddress.ip_address(address)
```

Comment:
I'm not sure this check is sufficient. For example `ipaddress.ip_address('0.0.0.0')` will not raise. One option would be to check the `.is_global` / `.is_private` attributes but then we'd need knowledge about which kind of IP we're expecting.